benchmarks/nemo/nemolite2d/psykal/Makefile

```diff
@@ -99,21 +104,23 @@ nemolite2d_hybrid: common_lib timer_lib mpi_inf_lib nemolite2d_alg.f90 omp_trans
 
 
 # OpenACC PSyclone code-generation of NemoLite2D
-nemolite2d_acc: common_lib timer_lib serial_inf_lib nemolite2d_alg.f90 acc_transform.py
+nemolite2d_acc: serial_libs nemolite2d_alg.f90 psyclone_scripts/acc_transform.py
```

Comment:
This one also fails but perhaps that's expected (I think you say it doesn't work in the PR)? If so please add a comment here and in the README.

Reply:
Added a comment and updated the readme. I think the script is alright, the fix need to be done in PSyclone.